assets/js/components/CleanUpButton/CleanUpButton.jsx

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
 
 import { EOS_CLEANING_SERVICES } from 'eos-icons-react';
+import classNames from 'classnames';
 
 import Button from '@components/Button';
 import Spinner from '@components/Spinner';
 
-function CleanUpButton({ cleaning, onClick }) {
+function CleanUpButton({ cleaning, onClick, className }) {
+  const buttonClasses = classNames(
+    'inline-block mx-0.5 border-green-500 border w-fit',
+    className
+  );
+
   return (
     <Button
       type="primary-white"
-      className="inline-block mx-0.5 border-green-500 border w-fit"
+      className={buttonClasses}
       size="small"
       disabled={cleaning}
       onClick={() => onClick()}
```

Comment:
Just a comment, when we have this "dummy" components, without any particular logic on the event handlers, for example take a modal which has on the `onClose` prop the id of something passed like `onClose(id)`, that is a complex handler.

In that button we have a direct onclick handler, without any logic and it's a common pratice, but totally optional, to pass directly the handler, like `onClick={onClick}`, so the entire event chain of the dom event is passed up to the parent. The parent can listen if want to the event as first argument of the onClick prop callback.

This is completely optional in that case, there are different opinions about that, for sure it's true in pure design systems component, take bootstrap for example, you want to give the callee the direct facade to the html event.

If you want change it, otherwise leave as is and we could have this comment as a future discussion.

Reply:
Actually, now that I read this, I'm wondering how it works (JS mysteries to me XD).
Because in the component, we have `() => onClick()`, but in the call to the component we have:
```
onClick={() => {
    openDeregistrationModal(item);
    openDeregistrationModal(item);
 }}
```
so it is kind of done twice.
Sorry for my JS dummy questions hehe
Maybe it is totally normal